lib/list.js

```diff
@@ -82,17 +82,17 @@ const COLLECTIONS_MAPPING = ['ds:3', 0, 1];
 const CLUSTER_URL_MAPPING = [0, 0, 3, 4, 2];
 const CLUSTER_NAME_MAPPING = [0, 1];
 const CLUSTER_NAMES = {
-  TOP_FREE: 'top free apps',
-  TOP_PAID: 'top paid apps',
-  GROSSING: 'top grossing apps',
-  TOP_FREE_GAMES: 'top free games',
-  TOP_PAID_GAMES: 'top paid games',
-  TOP_GROSSING_GAMES: 'top grossing games',
-  TRENDING: 'trending',
-  NEW_FREE: 'top new free android apps',
-  NEW_PAID: 'top new paid android apps',
-  NEW_FREE_GAMES: 'top new free games',
-  NEW_PAID_GAMES: 'top new paid games'
+  TOP_FREE: ['top free', 'top free apps'],
```

Comment:
can you explain this change better, from the perspective of the user of this library? why do we need TOP_FREE to include these two cases? shouldn't it also include 'top free games' for that matter?